### PR TITLE
docs: add Mike Huang to GDE resources

### PR DESCRIPTION
**Pull request by wellwind**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by wellwind** [comment]
@googlebot I signed it!

**Review by gabrielbussolo — COMMENTED** [review]
I verified and you are really you! :laughing: 
I saw @StephenFluin recommend images with <30kb in another [pull request
](https://github.com/angular/angular/pull/35925#issuecomment-610081966), so I wold like to give the same suggestion :grin:

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -835,5 +835,13 @@
     "bio": "Manu heads technical program management for Angular at Google. Manu keeps the big picture in focus and works with cross-functional teams to plan, execute and usher programs through the entire lifecycle.",
     "groups": ["Angular"],
     "lead": "juleskremer"
+  },
+  "mikehuang": {
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json:839]: Please use your GitHub handle as key 🙏

File: aio/content/marketing/contributors.json [hunk 2]
```diff
@@ -835,5 +835,13 @@
...
+    "name": "Mike Huang",
+    "picture": "mikehuang.jpg",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json:841]: Also, the extension does not match the actual image file extension :grin:

**Review by IgorMinar — APPROVED** [review]
I squashed the commits, rebased this, and force pushed into the branch. now it LGTM.

**Pull request closed without merging**